Soap/Exception/MissingClientException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/*
+* This file is part of the CleverAge/ProcessBundle package.
+*
+* Copyright (C) 2017-2018 Clever-Age
+*
+* For the full copyright and license information, please view the LICENSE
+* file that was distributed with this source code.
+*/
+
+namespace CleverAge\ProcessBundle\Soap\Exception;
+
+use CleverAge\ProcessBundle\Exception\ProcessExceptionInterface;
+
+/**
+ * Exception thrown when trying to fetch a missing Soap client
+ *
+ * @author Madeline Veyrenc <mveyrenc@clever-age.com>
+ */
+class MissingClientException extends \UnexpectedValueException implements ProcessExceptionInterface
+{
+    /**
+     * @param string $code
+     */
+    public function __construct($code)
```

Comment:
It's best not to change the default constructor of the base exception to allow the injection of the previous error if needed.
Setting a default message can be done with a static constructor method.

Reply:
So fix need to be apply on MissingTransformerException

Reply:
and on other Exception of the process bundle